README.md

```diff
@@ -12,32 +15,6 @@ with Ecto, but is configurable to work with any persistence library.
 
 ## Installation
 
-#### To install in all environments (useful for generating seed data in dev/prod):
-
-In `mix.exs`, add the ExMachina dependency:
-
-```elixir
-def deps do
-  # Get the latest from hex.pm. Works with Ecto 3.0
-  [
-    {:ex_machina, "~> 2.7.0"},
-  ]
-end
-```
-
-And start the ExMachina application. For most projects (such as
-Phoenix apps) this will mean adding `:ex_machina` to the list of applications in
-`mix.exs`. You can skip this step if you are using Elixir 1.4 or later.
-
-```elixir
-def application do
-  [mod: {MyApp, []},
-   applications: [:ex_machina, :other_apps...]]
-end
-```
-
-#### Install in just the test environment with Phoenix:
-
 In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
```

Comment:
Wrap this up in the release please update tag so the version always matches latest.